crates/sui-indexer-alt-consistent-store/src/metrics.rs

```diff
@@ -0,0 +1,345 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use prometheus::{register_int_gauge_vec_with_registry, IntGaugeVec, Registry};
+use rocksdb::{properties, properties::num_files_at_level, AsColumnFamilyRef};
+use std::sync::Arc;
+use std::time::Duration;
+use tracing::{debug, error, warn};
+
+const METRICS_ERROR: i64 = -1;
+
+pub struct ConsistentStoreMetrics {
```

Comment:
Why do we have the two level nesting here? e.g. could we have all the metrics just directly on this type?